### DISCUSSION TITLE
Google Cloud Tasks with Google App Engine: 14 UNAVAILABLE: read ECONNRESET

### DISCUSSION THREAD

**Discussion by AllanOliveiraM** [googleapis/google-cloud-node#5799] [Q&A]
Hi everyone, how's it going?

I need some help with the Cloud Tasks client library. I have an application running on App Engine, and I use Cloud Tasks to queue the processing of some critical application data.

I'm using the `createTask` method to do this, and everything works normally most of the time. The problem is that occasionally, a task cannot be created, and I encounter a "14 UNAVAILABLE: read ECONNRESET" error in the logs.

I believe the issue is not with the API itself, but with my client library configuration.

I found [this issue](https://github.com/grpc/grpc-node/issues/1907) in the grpc-node GitHub repository, but I'm still having trouble configuring the client correctly.

If anyone could help me understand what I’m doing wrong, I’d be very grateful :)

![image](https://github.com/user-attachments/assets/fe6f9248-7d76-48d2-a8ee-2d1840469d96)

I don’t think the error is here, but I’ll include it as context.

![image](https://github.com/user-attachments/assets/a0bd88bf-0f6d-414e-8a07-2048071db1b6)
![image](https://github.com/user-attachments/assets/70cb7c41-b99c-429f-ba6b-09711f70b883)

**Comment by omar-dev-amrk** [comment]
I'm having the same problem. Have you found a solution or why it happens since then?
